docs/src/main/sphinx/admin/properties-writer-scaling.rst

```diff
@@ -39,8 +39,9 @@ per-query basis using the ``writer_min_size`` session property.
 * **Type:** ``boolean``
 * **Default value:** ``true``
 
-Enable preferred write partitioning. When set to ``true`` each partition is
-written by a separate writer. As a result, for some connectors such as the
+Enable preferred write partitioning. When set to ``true`` and sufficient partitions
+are being written (see ``preferred-write-partitioning-min-number-of-partitions``)
```

Comment:
and more than the minimum number of partitions, set in ``preferred-write-partitioning-min-number-of-partitions``, are created, each partition is written by a separate writer.

Reply:
changed. Is it ok now?